### DISCUSSION TITLE
Problem with artifact

### DISCUSSION THREAD

**Discussion by FrancisWaitack** [docker/build-push-action#1077] [Q&A]
Hello,

First thanks for your work, it saves lot of times.
But i have a problem with artifact, i have seen this issue : https://github.com/docker/build-push-action/issues/592#inputs and follow the steps but can't make it work...

Here is my workflow : 
`

    steps:
      - name: Clone repository
        uses: actions/checkout@v4

      - name: Set up QEMU
        uses: docker/setup-qemu-action@v1

      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v1

      - name: Log in to Registry
        uses: docker/login-action@v3
        with:
          registry: ${{ secrets.OVH_REGISTRY }}
          username: ${{ secrets.OVH_USERNAME }}
          password: ${{ secrets.OVH_TOKEN }}

      - name: Download artifact
        id: download-artifact
        uses: dawidd6/action-download-artifact@v3
        with:
          github_token: ${{secrets.GITHUB_TOKEN}}
          workflow: .github/workflows/dev_merge.yml
          branch: dev
          name: dev-artifact
          path: tmp_artifact

      - name: Build and push Docker image
        uses: docker/build-push-action@v5
        with:
          context: .
          file: ./docker/php-fpm/Dockerfile_gha
          push: true
          tags: ${{ secrets.OVH_REGISTRY }}/helm/php_fpm:dev_latest
`

and my docker file : 
`
WORKDIR /var/www/symfony
COPY tmp_artifact /var/www/symfony`

**Comment by crazy-max** [comment]
> `uses: dawidd6/action-download-artifact@v3`

Not sure what this action is but you should use the official one: https://github.com/actions/download-artifact
